tidb-configuration-file.md

```diff
@@ -342,6 +350,7 @@ Configuration items related to performance.
 + When the memory usage alarm is enabled, if [`server-memory-quota`](/tidb-configuration-file.md#server-memory-quota-new-in-v409) is not set, then the threshold of memory usage is ```the `memory-usage-alarm-ratio` value * the system memory size```; if `server-memory-quota` is set to a value greater than 0, then the threshold of memory usage is ```the `memory-usage-alarm-ratio` value * the `server-memory-quota` value```.
 + When TiDB detects that the memory usage of the tidb-server instance exceeds the threshold, it considers that there might be a risk of OOM. Therefore, it records ten SQL statements with the highest memory usage, ten SQL statements with the longest running time, and the heap profile among all SQL statements currently being executed to the directory [`tmp-storage-path/record`](/tidb-configuration-file.md#tmp-storage-path) and outputs a log containing the keyword `tidb-server has the risk of OOM`.
 + The value of this configuration item is the initial value of the system variable [`tidb_memory_usage_alarm_ratio`](/system-variables.md#tidb_memory_usage_alarm_ratio).
++ Since v6.1.0, the threshold is determined by configuration [`instance.tidb_memory_usage_alarm_ratio`](/tidb-configuration-file.md#tidb_memory_usage_alarm_ratio) or system variable [`tidb_memory_usage_alarm_ratio`](/system-variables.md#tidb_memory_usage_alarm_ratio). `memory-usage-alarm-ratio` still takes effect. But if `memory-usage-alarm-ratio` and `instance.tidb_memory_usage_alarm_ratio` are setting at the same time, TiDB will apply the value of `instance.tidb_memory_usage_alarm_ratio`.
```

Comment:
```suggestion
+ Since v6.1.0, this threshold is determined by the TiDB configuration item [`instance.tidb_memory_usage_alarm_ratio`](/tidb-configuration-file.md#tidb_memory_usage_alarm_ratio) or the system variable [`tidb_memory_usage_alarm_ratio`](/system-variables.md#tidb_memory_usage_alarm_ratio). `memory-usage-alarm-ratio` still takes effect. But if `memory-usage-alarm-ratio` and `instance.tidb_memory_usage_alarm_ratio` are set at the same time, TiDB will apply the value of `instance.tidb_memory_usage_alarm_ratio`.
```